developer-docs-site/static/move_demo/sources/CAPMessageBoard.move

```diff
@@ -0,0 +1,202 @@
+/// This module demonstrates a basic messageboard using capability to control the access.
+/// admin can
+///     (1) create its messageboard "CapMessageBoard"
+///     (2) offer participants capability to update the pinned message
+///     (3) remove the capability from a participant
+/// participant can
+///     (1) register for the board
+///     (2) redeem the offered capability to update pinned message
+///     (3) submit a new message
+///
+/// The module also emits two types of events for subscribes
+///     (1) notice cap changes event, this event contains the board address and participant offered capability
+///     (2) noice change event, this event contains the board, message and message author
+module Examples::CapMessageBoard {
+    use Std::Vector;
+    use Std::Signer;
+    use Std::Offer;
+    use Std::Event::{Self, EventHandle};
+
+    // Error map
+    const EACCOUNT_NO_NOTICE_CAP: u64 = 1;
+    const EONLY_ADMIN_CAN_REMOVE_NOTICE_CAP: u64 = 2;
+
+    struct CapMessageBoard has key {
+        notice: vector<u8>
+    }
+
+    /// provide the capability to alert the board message
+    struct NoticeChangeCapability has key, store {
+        board: address
+    }
+
+    struct NoticeCapEventHandle has key {
+        change_events: EventHandle<NoticeCapEvent>
+    }
+
+    struct NoticeCapEvent has store, drop {
```

Comment:
probably needs a boolean `added_not_removed` or something like hat to distguish what has happened

Reply:
update the event name to MessageCapUpdateEvent to clearly indicate allowed addr is updated.